meetings/2022-05-09.md

```diff
@@ -0,0 +1,61 @@
+# W3C Solid Community Group: Test Suite Panel
+
+* Date: 2022-05-09T14:00:00Z
+* Call: https://meet.jit.si/solid-test-suite
+* Chat: https://gitter.im/solid/test-suite
+* Repository: https://github.com/test-suite-panel
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: The Test Suite Panel is a "horizontal" panel. We need to revisit/clarify our understanding of the Test Suite Panel alongside other panels in the CG as well as contributors to specs and implementers.
+
+
+### Test Suite Panel Charter
```

Comment:
MJ: Provide tools for server implementers that help them conform with the spec.
MJ: Provide information about pod providers that reports which ones conform with the spec.
MJ: Provide information for self-hosters that helps them choose an open source implementation that conforms with the spec.
MJ: Provide information for Solid app developers that helps them understand which pod providers and which open source implementations their app will be compatible with.
MJ: Make sure this information is up to date and accurate. When in doubt, err on the cautious side.
MJ: Make sure this information is findable. Currently neither our tools nor our reports are findable through solidproject.org, and that is a problem that requires our attention.

Reply:
>MJ: Provide tools for server implementers that help them conform with the spec.

This can be a deliverable.

>MJ: Provide information about pod providers that reports which ones conform with the spec.

Covered by PROPOSAL-1-3.

>MJ: Provide information for self-hosters that helps them choose an open source implementation that conforms with the spec.
>MJ: Provide information for Solid app developers that helps them understand which pod providers and which open source implementations their app will be compatible with.

Covered by PROPOSAL-1 in that applications can be built to use the data provided by authoritative reports and . However, this is not for the Charter.

>MJ: Make sure this information is up to date and accurate. When in doubt, err on the cautious side.

Covered by PROPOSAL-1-3 in that information reflects what's tested, when, how... and all the relevant metadata. Newer reports can be submitted and we can figure out the process for that too (PROPOSAL-1).

>MJ: Make sure this information is findable. Currently neither our tools nor our reports are findable through solidproject.org, and that is a problem that requires our attention.

Not a Charter issue but I agree. For starters, specifications will link to tests suites - there is an issue for this already.